### PR TITLE
Trend revamp

### PR DESCRIPTION
**Pull request by jvillacampa**
Changed palette, layout of ui, legend. Added option to show numerator. Simplifying download chart.

### CODE REVIEW

File: server.R [hunk 1]
```diff
@@ -1062,22 +1062,23 @@ showModal(welcome_modal)
 # Reactive controls
   #Controls for chart. Dynamic selection of locality and iz.
   output$loc_ui_trend <- renderUI({
-    selectInput("locname_trend", "HSC locality", 
+    selectizeInput("locname_trend", "HSC locality",
```
Comment (vaelliott) [inline server.R]: just wondering what is the difference between Selectize and Select is?
selectize=TRUE does what?

File: ui.R [hunk 2]
```diff
@@ -363,34 +363,37 @@ tabPanel("Barcode", icon = icon("barcode"), value = "barcode",
...
                           selectInput("indic_trend", "Indicator", choices=indicator_list),
-                          awesomeCheckbox("ci_trend", label = "95% confidence intervals", value = FALSE),
-                          awesomeCheckbox("colorblind_trend",  
-                                          label = "Improved accessibility", value = FALSE),
-                          actionButton("defs_trend",label="Definitions", icon= icon('info'), class ="down"),
+                          div(title= "Display the rate/percentage data or the raw numbers.",
+                              awesomeRadio("var_plot_trend", label =NULL, inline = TRUE, 
+                                            choices = c("Rate/Percentage" = "measure", 
+                                                   "Numerator" = "numerator"))),
+                          div(title="Show or hide the 95% confidence intervals for the data selected.", # tooltip
```
Comment (vaelliott) [inline ui.R]: What about making the 95% CI button be conditional on rate/percentage being selected?
might save confusion?

File: server.R (hunk 1 above)
Comment (jvillacampa) [reply server.R]: Very little difference. This shows it with examples: https://shiny.rstudio.com/gallery/selectize-vs-select.html

File: ui.R (hunk 2 above)
Comment (jvillacampa) [reply ui.R]: Good point, done.

**Comment by jvillacampa** [comment]
I disabled the cis when they are not available. The save chart works fine for me. Any example of when it doesn't work?

**Comment by jvillacampa** [comment]
Nevermind, I have found the issue with the save chart

**Comment by vaelliott** [comment]
damn I did things in the wrong order and approved the changes...
I still have problems with the chart download - COPD indicator one example but actually i think its any chart.  I have attached a screen shot of the error, its complaining about PhantomJS.  Is PhantomJS a package that in't in the global script?

image

**Comment by jvillacampa** [comment]
Yeah, sorry I was working on fixing it, now it should work for anyone I hope

**Comment by vaelliott** [comment]
Perfect, that works now.
Starting to see how this Github reviewing will be really helpful.

File: ui.R (hunk 2 above)
Comment (vaelliott) [reply ui.R]: looks good

**Pull request merged**